### PR TITLE
Create 052123-tworeviews.md

### PR DESCRIPTION
**Pull request by dlang3**
The third ADR to be committed (requiring two pull requests)

### CODE REVIEW

**Review by matty-zee — APPROVED** [review]
looks good to me 👍

File: specs/adrs/052123-tworeviews.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Use 2 Reviews for any Pull Request to Merge
+
+## Context and Problem Statement
+
+How many reviews needed for merge?
```
Comment (nb110240) [inline specs/adrs/052123-tworeviews.md:5]: 2

**Review by nb110240 — APPROVED** [review]
looks right

**Comment by nb110240** [comment]
👍🏻

**Pull request merged**